invenio_communities/assets/semantic-ui/js/invenio_communities/settings/submissionPolicy/SubmissionPolicyForm.js

```diff
@@ -53,14 +46,44 @@ ReviewPolicyField.defaultProps = {
   label: "",
 };
 
-export class CurationPolicyForm extends Component {
+const RecordSubmissionPolicyField = ({ label, formConfig, ...props }) => {
+  const [field] = useField(props);
+  const fieldPath = "access.record_submission_policy";
+  const { record_submission_policy: subPolicy } = formConfig.access;
+  return (
+    <>
+      {subPolicy.map((item) => (
+        <React.Fragment key={item.value}>
+          <RadioField
+            key={item.value}
+            fieldPath={fieldPath}
+            label={item.text}
+            labelIcon={item.icon}
+            checked={_get(field.value, fieldPath) === item.value}
+            value={item.value}
+          />
+          <label className="helptext">{item.helpText}</label>
+        </React.Fragment>
+      ))}
+    </>
+  );
+};
+
+RecordSubmissionPolicyField.propTypes = {
+  label: PropTypes.string,
+  formConfig: PropTypes.object.isRequired,
+};
+
+RecordSubmissionPolicyField.defaultProps = {
+  label: "",
+};
+
+class SubmissionPolicyForm extends Component {
```

Comment:
```suggestion
class RecordSubmissionPolicyForm extends Component {
```

sorry 🙃

Reply:
The form still should be called `SubmissionPolicy` - this is the whole form that includes Review policy and Record submission policy